src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -38,4 +38,42 @@
       expect(response).to redirect_to(user_show_path(User.current))
     end
   end
+
+  describe "POST #delete_request" do
+    before do
+      login(submitter)
+    end
+
+    context "a valid request" do
+      before do
+        post :delete_request, project: target_project, package: target_package, description: "delete it!"
+      end
+
+      it { expect(response).to redirect_to(request_show_path(number: bs_request)) }
+      it { expect(flash[:success]).to match("Created .+repository delete request #{bs_request.number}") }
```

Comment:
because it's a regex. I don't know the created request number beforehand. Also the 'repository delete request' is a link. So there are some 'a' tags I did not want to test.